src/www/ui/change-license-processPost.php

```diff
@@ -95,20 +95,34 @@ function Output()
     $decisionMark = @$_POST['decisionMark'];
     if(!empty($decisionMark))
     {
-      $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
-      /** @var ItemTreeBounds */
-      $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
-      $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
-      if (empty($errMsg))
-      {
-        return new JsonResponse(array('result'=>'success'));
+      if(!is_array($itemId)) {
```

Comment:
please try to make it like https://github.com/fossology/fossology/wiki/Coding-Style